tests/commands/test_cmd_config.py

```diff
@@ -0,0 +1,73 @@
+# third-party imports
+from click.testing import CliRunner
+
+# local imports
+from shpkpr.cli import cli
+
+
+def test_no_args():
+    runner = CliRunner()
+    result = runner.invoke(cli, ['config'])
+
+    assert result.exit_code == 0
+    assert 'Usage:' in result.output
```

Comment:
Seems good, I just wanted to make sure they had different intentions.

For what it's worth, if these are intended to be more like unit tests I would consider moving them to a single assertion per test. Multiple assertions lends itself more to the integration test style where it's expensive to go through the whole stack each time while single assertion unit tests help ensure that a test will only ever fail for one reason which can make future refactors easier, especially when you're trying to figure out what the intention of the test was.